src/classes/CSVReaderTest.cls

```diff
@@ -0,0 +1,151 @@
+/**
+* File Name :   CSVReaderTest.cls
+* Description : Made to only test the methods in CSVReader.cls
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 07/04/2014 Klea Kolaric   Initial version
+*
+*
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.